net-misc/zerotier/zerotier-1.4.6.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
Please when updating remember to update the year, I know it was contributed before 2020, but now when making changes.